### ISSUE TITLE
[DOXIA-348] Doxia 1.1.2-SNAPSHOT won't build with a clean repository

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia#353]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** opened **[DOXIA-348](https://issues.apache.org/jira/browse/DOXIA-348?redirect=false)** and commented

Hi

I deleted my repository to test that 1.1.2-SNAPSHOT would build for others. But:

Downloading: http://repo1.maven.org/maven2/org/apache/maven/doxia/doxia-decoration-model/1.0-alpha-8/doxia-decoration-model-1.0-alpha-8.jar
39K downloaded  (doxia-decoration-model-1.0-alpha-8.jar)
520K downloaded  (bcel-5.2.jar)
[INFO] [clirr:check {execution: default}]
[INFO] Comparing to version: 1.1
Downloading: http://repo1.maven.org/maven2/org/apache/maven/doxia/doxia/1.1/doxia-1.1.pom
14K downloaded  (doxia-1.1.pom)
[INFO] ------------------------------------------------------------------------
[ERROR] FATAL ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Cannot open /Users/xxx/.m2/repository/org/apache/maven/doxia/doxia/1.1/doxia-1.1.pom for reading
error in opening zip file
[INFO] ------------------------------------------------------------------------
[INFO] Trace
net.sf.clirr.core.CheckerException: Cannot open /Users/nsowatsk/.m2/repository/org/apache/maven/doxia/doxia/1.1/doxia-1.1.pom for reading
at net.sf.clirr.core.internal.bcel.BcelTypeArrayBuilder.createClassSet(BcelTypeArrayBuilder.java:73)
at org.codehaus.mojo.clirr.AbstractClirrMojo.resolvePreviousReleaseClasses(AbstractClirrMojo.java:306)
at org.codehaus.mojo.clirr.AbstractClirrMojo.executeClirr(AbstractClirrMojo.java:192)
at org.codehaus.mojo.clirr.AbstractClirrMojo.executeClirr(AbstractClirrMojo.java:182)
at org.codehaus.mojo.clirr.ClirrCheckMojo.execute(ClirrCheckMojo.java:68)
at org.apache.maven.plugin.DefaultPluginManager.executeMojo(DefaultPluginManager.java:483)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoals(DefaultLifecycleExecutor.java:678)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoalWithLifecycle(DefaultLifecycleExecutor.java:540)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoal(DefaultLifecycleExecutor.java:519)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoalAndHandleFailures(DefaultLifecycleExecutor.java:371)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeTaskSegments(DefaultLifecycleExecutor.java:332)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.execute(DefaultLifecycleExecutor.java:181)

---

**Affects:** 1.2

**Comment by jira-importer** [comment]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** commented

A work around is to build doxia 1.1 locally, then mvn clean 1.1.2, then the build works.

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

I cannot reproduce this (Linux, Maven 2.1, java 6). I deleted my local repo, then 'mvn clean install' worked fine on current trunk.

The error message you get is similar to MCLIRR-12, but this was fixed in clirr-plugin-2.2.1 and we are using 2.2.2. Anyway, since the error comes from the clirr plugin, another workaround should be to just disable clirr for the build.
